### ISSUE TITLE
sign error in atleast 2 saved parks

### ISSUE THREAD

**Issue by jamtraks** [OpenRCT2/OpenRCT2#8033]
-->
**OS:** windows vista 32-bit
**Version:** e.g.0.2.1-180
**Commit/Build:** 2ccc183 on develop

after opening 2 saved parks of mine that had been save before the error occurred, i noticed that some but not all signs now say "sign" instead of what was intended. If i right click the sign to edit it, the game crashes

- [did not test
n/a

**Steps to reproduce:**
1. open park and observe signs
2. 

**Dump file**

**Screenshots / Video:**

![sign error](https://user-images.githubusercontent.com/17103908/46249104-7dce3200-c3f1-11e8-8e92-a93714400a19.jpg)

**Save game:**

[Bumbly Bazaar by Josh Marsilio 2.96.sv6.zip](https://github.com/OpenRCT2/OpenRCT2/files/2430928/Bumbly.Bazaar.by.Josh.Marsilio.2.96.sv6.zip)

**Comment by Broxzier** [comment]
I noticed this too in another saved game. It looks like they are not being saved correctly. There are warning being printed in the console when attempting to place large scenery objects.

```
WARNING[d:\data\projects\openrct2\src\openrct2\object\objectmanager.cpp:68 
    (ObjectManager::GetLoadedObject)]:
    Object index 771 exceeds maximum for type 2.
```

**Comment by jamtraks** [comment]
Yes, this error is occurring in several saves i have, but when i test those parks in my non-launcher (older version of openrct2) the error does not occur.
I do not understand what you are saying though, are you suggesting i saved the error into the park? and that the save file is damaged?

**Comment by Broxzier** [comment]
If you're able to load and use them in the older save, then no, that's probably not the case. What is the version of the older build you're using?

**Comment by jamtraks** [comment]
.... i had to wait 5 minutes for the older version of openrct2 to load just now, anyway that older version is 0.2.1-34 a1fd033, the sign problem does not occur in that version.

**Comment by janisozaur** [comment]
you can extract OpenRCT2's version information the usual way, by passing `--version`

**Comment by jamtraks** [comment]
? @janisozaur 

> you can extract OpenRCT2's version information the usual way, by passing --version
is that message to me?

**Comment by jamtraks** [comment]
who is "you" are you saying that to me?

**Comment by Gymnasiast** [comment]
I can reproduce it.

**Comment by jshute** [comment]
I'm having this same issue. Some 3D signs are linking to a different sign. Deleting doesn't work and sometimes causes a game crash. At first I thought it was some sort of overrun with the banner limit I confirmed in the console that I'm not near the banner limit. It's happening in both the release and develop versions. Happening in develop `df768a3` and stable `8ac731e`.

**Comment by deurklink** [comment]
I noticed the same in most of my older saves. They used to display fine, as I can see in screenshots I took back then, but currently they display the wrong texts.

Old (giant) screenshot:
![round_6](https://user-images.githubusercontent.com/6469413/48375064-fac61a00-e6c6-11e8-98ea-9ef626e64061.png)

Screenshot I just took, in the lower corner:
![image](https://user-images.githubusercontent.com/6469413/48375188-6f00bd80-e6c7-11e8-873f-ea387ca5b242.png)

Here's the save:

[Multiplayer_park_-_Round_6.zip](https://github.com/OpenRCT2/OpenRCT2/files/2573605/Multiplayer_park_-_Round_6.zip)

**Comment by deurklink** [comment]
I did not overwrite the save since creating that screenshot.. What's interesting is that when you right click one of the wrong signs it actually takes you to another sign:

![image](https://user-images.githubusercontent.com/6469413/48376492-6316fa80-e6cb-11e8-82b6-83b589a37c86.png)

**Comment by deurklink** [comment]
@Broxzier The newest version did not resolve the problem for me.

**Comment by deurklink** [comment]
I did some more testing.

Placing more than 63 signs crashes game to desktop. 
![test park 2018-11-19 23-03-16](https://user-images.githubusercontent.com/6469413/48738176-6fadcc80-ec50-11e8-88ac-2f03430e9638.png)

I did not count them, but the parks which have these problems all have a lot of signs, probably more than 63.

Also, if you try to place a sign in one of these parks, the ghost will look like this:

![image](https://user-images.githubusercontent.com/6469413/48738280-c3b8b100-ec50-11e8-9ab8-9e942ffd3dda.png)

Here's another save which has the problem:
[Multiplayer_park_-_Round_8.zip](https://github.com/OpenRCT2/OpenRCT2/files/2597214/Multiplayer_park_-_Round_8.zip)

**Comment by deurklink** [comment]
Maybe this is of any help? This is what I get when I try to place a 64th sign:

![image](https://user-images.githubusercontent.com/6469413/49399324-48232d80-f741-11e8-8a66-b1c582a68093.png)

**Comment by deurklink** [comment]
Thank you so much!

**Comment by deurklink** [comment]
Ah never mind, signs in old parks are still messed up and referring to other signs..

**Comment by Gymnasiast** [comment]
Old parks are shot, indeed.

**Comment by Gymnasiast** [comment]
Shit, I forgot the S6 import code :/
Should be fixed now.
